app/index.js

```diff
@@ -103,6 +103,9 @@ app.on('ready', () => {
 
     win.loadURL(url);
 
+    // Make sure this newly opened window is seen to now have focus
+    win.focusTime = process.uptime();
```

Comment:
The idea is perfect :) 
However it's a risky design choice in my opinion. Maybe it would be better to just trigger win.focus() (if it works) or encapsulate both this `win.focusTime = process.uptime();` and the one at line 287 in a function called `registerFocusTime` (or something similar).
So, if in a future someone will patch how the focus time is stored, it will only to change that function and `getLastFocusedWindow()`.

My 2 cents ;)